roles/servicetelemetry/defaults/main.yml

```diff
@@ -3,8 +3,7 @@
 #
 # remove SmartGateway object when cloud no longer in current `clouds` object list.
 clouds_remove_on_missing: false
-# default observability strategy (compatible with STF 1.3)
-observability_strategy: use_community
+observability_strategy: use_redhat
```

Comment:
Oh that's a good point - I'd assumed all deployed instances would have `observability_strategy: use_community` since that's the previous default, and that they wouldn't change when you upgrade. If this will change an existing STF deployment from use_community to use_redhat, that's bad news and was not the intention. I will test this, thanks.

Reply:
I tested this and it's true that it will attempt to convert existing installs. We either need to make that a smooth experience, or figure out another strategy for the migration. For the time being I'm going to revert the default while we think this through.

Reply:
Update on this thread is that I took care of these issues with [0838da7](https://github.com/infrawatch/service-telemetry-operator/pull/421/commits/0838da71b79afd51b7bd245397abb34e3621ba85) and show some testing below https://github.com/infrawatch/service-telemetry-operator/pull/421#issuecomment-1526167871